packages/web/components/complex/all-pools-table-v2.tsx

```diff
@@ -34,11 +38,67 @@ import { AprDisclaimerTooltip } from "../tooltip/apr-disclaimer";
 
 type Pool =
   RouterOutputs["edge"]["pools"]["getMarketIncentivePools"]["items"][number];
-type SortKey = MarketIncentivePoolSortKey | undefined;
 /** UI doesn't support cosmwasm pools as first class so exclude it from list of filter options. */
 type PoolTypeFilter = Exclude<Pool["type"], "cosmwasm">;
 type PoolIncentiveFilter = NonNullable<Pool["incentiveTypes"]>[number];
 
+const poolTypes = [
+  "weighted",
+  "stable",
+  "concentrated",
+  "cosmwasm-transmuter",
+] as const;
+
+const marketIncentivePoolsSortKeys = [
+  "totalFiatValueLocked",
+  "feesSpent7dUsd",
+  "feesSpent24hUsd",
+  "volume7dUsd",
+  "volume24hUsd",
+  "aprBreakdown.total",
+] as const;
```

Comment:
Would it be possible to import these from the server?

Reply:
We can share them from a common folder, but we can't import them from the server folder because when you do this it also imports the trpc initializer and gives error